src/layouts/shortcodes/gsctl_deprecation_disclaimer.html

```diff
@@ -0,0 +1 @@
+<div class="well disclaimer"><p>gsctl and the <a href="/ui-api/rest-api/">REST API</a> are being phased out. We don't have an end-of-life date yet. However, we recommend to familiarize yourself with our <a href="/ui-api/management-api/">Management API</a> and the <a href="/ui-api/kubectl-gs/">kubectl gs</a> plug-in as a future-proof replacement.</p></div>
```

Comment:
```suggestion
<div class="well disclaimer"><p>gsctl and the <a href="/ui-api/rest-api/">REST API</a> are being phased out. We don't have an end-of-life date yet. However, we recommend to familiarize yourself with our <a href="/ui-api/management-api/">Management API</a> and the <a href="/ui-api/kubectl-gs/">kubectl gs</a> plugin as a future-proof replacement.</p></div>
```

Reply:
hm just learned that most dictionaries use 'plug-in' instead of 'plugin', however, 'plugin' is generally accepted in software development.
And since we are using 'plugin' in all our documentation, I would prefer to stay consistent but no strong opinion :)

Reply:
hm.. just learned that most dictionaries use 'plug-in' as a correct writing but 'plugin' is generally accepted in software development. 
Since in all our existing documentation we already use 'plugin', I would prefer to stay consistent, but no strong opinion.

Reply:
Interesting. I'd say it's inconsistent currently, so we'd have to align this throughout our docs anyway.